### DISCUSSION TITLE
Missing tooltips on sidebar

### DISCUSSION THREAD

**Discussion by auribuo** [zen-browser/desktop#2489] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Some buttons on the Zen Sidebar do not have tooltips on hover. 
This affects Zen-only buttons. All buttons also present in Firefox do not have this issue.

Missing tooltips:
![image](https://github.com/user-attachments/assets/1219d102-a850-49c6-a854-fba03fda9649)

![image](https://github.com/user-attachments/assets/aba01ce8-c0eb-41e7-9920-d4aec44a5f59)

Buttons also present in Firefox work as expected:
![image](https://github.com/user-attachments/assets/bf43c847-4112-4326-b713-e532910fd7c9)

The expected behavior would be for the buttons to have a tooltip explaining the function and to include the keyboard shortcut, if applicable.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.4

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1715/missing-tooltips-on-sidebar">ZEN-1715 Missing tooltips on sidebar</a></p>

**Comment by salamijason** [comment]
Hi! I think this is less of a bug and more of just something that the devs did not think of. Will be moving this over to discussions as a suggestion. thank you for bringing this up tho! I hadn't realized the Zen buttons didnt have tooltips.

**Comment by alefnull** [comment]
just found this and thought i'd add my $0.02, as i actually just installed Zen for the first time last night, and so far this has been the only thing that's bothered me (and to be fair, it's only a slight bother and not a deal breaker because so far this is a great browser and i'm already setting it up for daily use).

specifically, i like to keep the tab sidebar in the more compact mode where it only shows the icons, but when doing that, i don't get any tooltips when hovering over the tab icons, so if i have say multiple youtube tabs open at once, i can't differentiate between them without clicking on them or expanding the tab bar to see the titles.

**Comment by jpumfrey** [comment]
It is likely this one got missed as most of them do work for the other buttons. It _should_ be standard practice to include these tooltips for accessibility but, given Zen is in Alpha, I suspect improvements to accessibility will be addressed as things become more polished. 

Here's a good resource to help understand their importance:
https://www.a11y-collective.com/blog/tooltips-in-web-accessibility/

**Comment by alefnull** [comment]
i should note that in the 15 minutes since i posted my comment, i've disabled a couple of "mods" i was trying out, changed some other Zen settings, and restarted the browser, and i'm now getting not only tooltips with the titles but also a preview of the tab.

so i suppose my $0.02 was more like $0.00...
